pkg/ddl/constant.go

```diff
@@ -63,10 +63,12 @@ const (
 		state_update_time bigint,
 		end_time TIMESTAMP,
 		meta longblob,
+		ordinal int,
```

Comment:
Could we add a comment to describe the ordinal field usage?

Reply:
see `Subtask.Ordinal`